completion/available/gradle.completion.bash

```diff
@@ -31,31 +32,32 @@ function __gradle-set-project-root-dir() {
 
 __gradle-init-cache-dir() {
 	cache_dir="$HOME/.gradle/completion"
-	mkdir -p $cache_dir
+	mkdir -p "$cache_dir"
 }
 
 __gradle-set-build-file() {
 	# Look for default build script in the settings file (settings.gradle by default)
 	# Otherwise, default is the file 'build.gradle' in the current directory.
 	gradle_build_file="$project_root_dir/build.gradle"
 	if [[ -f "$project_root_dir/settings.gradle" ]]; then
-		local build_file_name=$(grep "^rootProject\.buildFileName" "$project_root_dir/settings.gradle" \
+		local build_file_name
+		build_file_name=$(grep "^rootProject\.buildFileName" "$project_root_dir/settings.gradle" \
 			| sed -n -e "s/rootProject\.buildFileName = [\'\"]\(.*\)[\'\"]/\1/p")
 		gradle_build_file="$project_root_dir/${build_file_name:-build.gradle}"
 	fi
 }
 
 __gradle-set-cache-name() {
 	# Cache name is constructed from the absolute path of the build file.
-	cache_name=$(echo $gradle_build_file | sed -e 's/\//_/g')
+	cache_name=$(echo "$gradle_build_file" | sed -e 's/\//_/g')
```

Comment:
```suggestion
	cache_name=${gradle_build_file//'/'/_}
```

Reply:
here again, the /g catches 2 or more occurrences, and that makes more sense to me.

Reply:
once again, I dare not argue with the master :)